src/main/java/menion/android/whereyougo/MainApplication.java

```diff
@@ -1,17 +1,17 @@
 /*
  * This file is part of WhereYouGo.
- * 
+ *
```

Comment:
I can't revert it. I tried it via god damn vim and it refuses to revert that change. Since you have push rights to collaborator branches please try to revert it yourself. I am out of ideas how to do it. I haven tried it via changing the git config lineendings but nothing works.

Reply:
I changed your PR using the web editor.

Reply:
The formatting changes are back.
You need to pull my changes first before starting with the next changes.